docs/testing-components/unit-testing/index.md

```diff
@@ -115,41 +115,29 @@ and outputs we can test basically any JavaScript function that returns a value.
 
 ## Testing React
 
-To test react we use [Enzyme](https://enzymejs.github.io/enzyme/) plus other tools you can find out about [here](../test-driven-development/index.md).
+To test react we use [React Testing Library](https://testing-library.com/docs/react-testing-library/intro) plus other tools you can find out about [here](../test-driven-development/index.md).
 
 ```js
-const Text = (props) => {
-	return <p>{props.content}</p>;
+const Text = ({content, ...rest}) => {
+	return <p {...rest}>{content}</p>;
 }
 
 test('Should contain text', () => { 
-	const subject = shallow( 
-		<Text content='sample' /> 
+	render( 
+		<Text data-testid="test-text" content='sample' /> 
 	);
 	
-	const expected = 'sample'; 
-	const actual = subject.text()
-	expect(actual).toBe(expected); 
+	const textElement = screen.getByTestId('test-text');
+	const expected = 'sample';
+	
+	expect(textElement).toHaveTextContent(expected); 
 });
 ```
 
-If you wish to learn more about Enzyme's library of functions look [here](https://github.com/enzymejs/enzyme).
-
-The three main parts about Enzyme that you need to know are it's rendering methods.
-
-### shallow()
-
-[Shallow](https://github.com/enzymejs/enzyme/blob/master/docs/api/shallow.md) is the virtual DOM representation. It will only render the component plus one level of children. This allows
-us to stay within the smaller confines of a component when testing.
-
-### mount()
-
-[mount](https://github.com/enzymejs/enzyme/blob/master/docs/api/mount.md) is the virtual DOM representation. It will render everything inside the component, including all nested children.
-This is a little beyond unit testing as you start to test the integration of a few components.
+If you wish to learn more about React Testing Library's library of functions look [here](https://testing-library.com/docs/react-testing-library).
```

Comment:
I can't access this link. https://testing-library.com/docs/react-testing-library. Please fix it.

Reply:
I removed this link because we use the same one(the fixed) only a couple of lines above in the same context.